common/src/api/external/mod.rs

```diff
@@ -573,6 +573,18 @@ pub struct IdentityMetadata {
     pub time_modified: DateTime<Utc>,
 }
 
+impl Default for IdentityMetadata {
+    fn default() -> Self {
+        Self {
+            id: Uuid::new_v4(),
+            name: "a".parse().unwrap(),
+            description: "".into(),
+            time_created: Utc::now(),
+            time_modified: Utc::now(),
+        }
+    }
+}
```

Comment:
Is this necessary? Having the name "a" doesn't seem right as a default

Reply:
removed in b3334342